CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Fixed
+- Post-processing wasn't working on VR
```

Comment:
This line is misleading, this PR fixes the first blit in stereo mode but doesn't fix the other VR issues.